src/detail-page/components/Tabs.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from 'react';
```

Comment:
If we rename our `/tabs-components` folder to simply `/tabs` we can move also this component to this folder :smile_cat: